src/Compiler/FSComp.txt

```diff
@@ -1037,7 +1037,8 @@ lexUnexpectedChar,"Unexpected character '%s'"
 1154,lexOutsideDecimal,"This number is outside the allowable range for decimal literals"
 1155,lexOutsideThirtyTwoBitFloat,"This number is outside the allowable range for 32-bit floats"
 1156,lexInvalidNumericLiteral,"This is not a valid numeric literal. Valid numeric literals include 1, 0x1, 0o1, 0b1, 1l (int/int32), 1u (uint/uint32), 1L (int64), 1UL (uint64), 1s (int16), 1us (uint16), 1y (int8/sbyte), 1uy (uint8/byte), 1.0 (float/double), 1.0f (float32/single), 1.0m (decimal), 1I (bigint)."
-1157,lexInvalidByteLiteral,"This is not a valid byte literal"
+1157,lexInvalidAsciiByteLiteral,"This is not a valid ASCII byte literal. Value must be < 128y."
+1157,lexInvalidTrigraphAsciiByteLiteral,"This is not a valid ASCII byte literal. Value should be < 128y.\nNote: In a future F# version this Warning will be promoted to Error!"
```

Comment:
The same comment about `128y` applies here as above. I also think it would be better not to use an exclamation point here.

Reply:
Thanks Brian, fixed in both cases.